vit-servicing-station-lib/src/server/settings/config.rs

```diff
@@ -280,6 +280,19 @@ impl FromStr for LogLevel {
     }
 }
 
+impl fmt::Display for LogLevel {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            Self::Disabled => write!(f, "disabled"),
+            Self::Error => write!(f, "error"),
+            Self::Warn => write!(f, "warn"),
+            Self::Info => write!(f, "info"),
+            Self::Debug => write!(f, "debug"),
+            Self::Trace => write!(f, "trace"),
+        }
+    }
+}
```

Comment:
This looks good, but what do you think of this idea?

Could we rely on the serde_json for this ? this would rarely change but just in case we add more levels (maybe some custom ones). 

```rust
impl fmt::Display for LogLevel {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
            write!(f, serde_json::to_string(&self).unwrap())
        }
    }
}
```

That way we can keep up the representation to match exactly how we serialize.

Reply:
Uhm, actually, thinking about this. It is clearer this way what we are doing, and it matches the FromStr implementation. So, never mind.